### PR TITLE
PSREDEV-827 - Pipelines data for dora dashboards

### PR DESCRIPTION
**Pull request by monhaque**
# Description:

Pipelines data for dora dashboards

## Ticket number:
[PSREDEV-827]

## Checklist:
- [ ] Is my change backwards compatible? Please include evidence
- [ ] I have tested this and added output to Jira Comment:
- [ ] Documentation added (link) Comment:

[PSREDEV-827]: https://govukverify.atlassian.net/browse/PSREDEV-827?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Pave1s — DISMISSED** [review]

**Comment by monhaque** [comment]
Needs a bit of work. so converting to draft for now

**Comment by monhaque** [comment]
Ready for review

File: dashboards/dev-platform/pipelines-list.csv [hunk 1]
```diff
@@ -1,3 +1,220 @@
...
+core-team,ipv-core@digital.cabinet-office.gov.uk,update-ecs-env-vars
+core-team,ipv-core@digital.cabinet-office.gov.uk,upload-assets
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,dl-cri-api-v1
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,dl-cri-front
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-api-v1
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-front
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-api
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,address-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,bav-ipv-thirdparty-stub
```
Comment (merlinc) [inline dashboards/dev-platform/pipelines-list.csv:85]: I think this should be Kiwi team

File: dashboards/dev-platform/pipelines-list.csv [hunk 2]
```diff
@@ -1,3 +1,220 @@
...
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,common-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,dl-third-party-stubs
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,fraud-third-party-stubs
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-hmrc-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,kbv-hmrc-cri-front
```
Comment (merlinc) [inline dashboards/dev-platform/pipelines-list.csv:93]: @gds-jbentham Do you want the KBV HMRCs as Lime team?

File: dashboards/dev-platform/pipelines-list.csv [hunk 3]
```diff
@@ -1,3 +1,220 @@
...
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,otg-hmrc-service
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,pa-third-party-stubs
```
Comment (merlinc) [inline dashboards/dev-platform/pipelines-list.csv:95]: @gds-jbentham Is this one of yours?

**Review by two-first-names — DISMISSED** [review]

File: dashboards/dev-platform/pipelines-list.csv [hunk 4]
```diff
@@ -1,3 +1,220 @@
 team,email,samstackname1
-CRI-Lime,cri-lime-team@digital.cabinet-office.gov.uk,ipv-cri-passport-api
-CRI-Lime,cri-lime-team@digital.cabinet-office.gov.uk,ipv-cri-passport-front
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,build-orch-be-deploy
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,build-rp-stub
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,experian-email-check
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,int-rp-stub
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,prod-rp-stub
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,stage-rp-stub
+auth-team,digital-identity-authentication@digital.cabinet-office.gov.uk,staging-orch-be-deploy
```
[the remark this answers is not in the document]
Comment (monhaque) [reply dashboards/dev-platform/pipelines-list.csv]: Changed ownership. Thanks @mrwilson

File: dashboards/dev-platform/pipelines-list.csv [hunk 5]
```diff
@@ -1,3 +1,220 @@
...
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-api-v1
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,fraud-cri-front
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-api
+cri-lime-team,cri-lime-observability@digital.cabinet-office.gov.uk,ipv-cri-passport-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,address-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,bav-ipv-thirdparty-stub
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,check-hmrc-cri-front
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,common-cri-api
+cri-orange-team,cri-orange-observability@digital.cabinet-office.gov.uk,dl-third-party-stubs
```
Comment (gds-jbentham) [inline dashboards/dev-platform/pipelines-list.csv:83]: This should be Lime

File: dashboards/dev-platform/pipelines-list.csv (hunk 1 above)
Comment (monhaque) [reply dashboards/dev-platform/pipelines-list.csv]: Done. thanks

File: dashboards/dev-platform/pipelines-list.csv (hunk 3 above)
Comment (monhaque) [reply dashboards/dev-platform/pipelines-list.csv]: Changed. Thanks

File: dashboards/dev-platform/pipelines-list.csv (hunk 5 above)
Comment (monhaque) [reply dashboards/dev-platform/pipelines-list.csv]: Changed. Thanks

File: dashboards/dev-platform/pipelines-list.csv (hunk 2 above)
Comment (monhaque) [reply dashboards/dev-platform/pipelines-list.csv]: Changed. Thanks

**Review by two-first-names — APPROVED** [review]

**Pull request merged**